themes/bootstrap3/js/record.js

```diff
@@ -34,6 +34,37 @@ function checkRequestIsValid(element, requestType, icon = 'place-hold') {
     });
 }
 
+function wayfinderPlacementLinkLookup(element) {
+  var url = VuFind.path + '/AJAX/JSON?' + $.param({
+    method: 'wayfinderPlacementLinkLookup',
+  });
+  $.ajax({
+    method: 'POST',
+    dataType: 'json',
+    cache: false,
+    url: url,
+    data: {
+      placement: element.dataset.location || ''
+    }
+  })
+    .done(function checkValidDone(response) {
+      if (response.data.status) {
+        let wayfinder_link = document.createElement('a');
```

Comment:
We prefer hiding the icon in the placeholder HTML. If that's not an option, there's an icon helper (`VuFind.icon('icon-name')`) available in JS as well (add the icon to finna-js-icons.phtml to make it available in JS).